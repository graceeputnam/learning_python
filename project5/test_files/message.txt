Congratulations. You are learning how to program!
