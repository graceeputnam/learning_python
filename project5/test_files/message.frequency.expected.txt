{'c': 0.02, 'o': 0.12, 'n': 0.08, 'g': 0.06, 'r': 0.1, 'a': 0.1, 't': 0.06, 'u': 0.04, 'l': 0.04, 'i': 0.04, 's': 0.02, '.': 0.02, ' ': 0.12, 'y': 0.02, 'e': 0.04, 'h': 0.02, 'w': 0.02, 'p': 0.02, 'm': 0.02, '!': 0.02, '\n': 0.02}
